### PR TITLE
Homework-22

### PR DESCRIPTION
**Pull request by OlGubkina**

### CODE REVIEW

File: tests/locators/hw22.spec.js [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+            await signUpBtn.click()
+            // Как эти три строки можно вынести в beforeEach? Почему ругается на /
+
+            const form = page.locator('app-signup-modal form')
+            const nameInput = form.locator('#signupName')
+
+            await nameInput.fill("111")
+            await nameInput.blur()
+
+            const inputNameTip1 = page.locator('.invalid-feedback > :nth-child(1)')
```
Comment (sttaran) [inline tests/locators/hw22.spec.js:66]: you don't need any numbers in your variable name

Comment (sttaran) [reply tests/locators/hw22.spec.js:66]: in all tests

Comment (sttaran) [reply tests/locators/hw22.spec.js:66]: if there is `inputNameTip1` I'd expect to see `inputNameTip2` at least

File: tests/locators/hw22.spec.js [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+            await signUpBtn.click()
+            // Как эти три строки можно вынести в beforeEach? Почему ругается на /
+
+            const form = page.locator('app-signup-modal form')
+            const nameInput = form.locator('#signupName')
+
+            await nameInput.fill("Anna Maria")
+            await nameInput.blur()
+
+            const inputNameTip1 = page.locator('.invalid-feedback > :nth-child(1)')
```
Comment (sttaran) [inline tests/locators/hw22.spec.js:81]: use more precise selectors

for example
```
#signupName + .invalid-feedback
```

File: tests/locators/hw22.spec.js [hunk 3]
```diff
@@ -0,0 +1,543 @@
+import {expect, test} from "@playwright/test"
+
+test.describe("Home work locators", () => {
+    test.beforeEach(async () => {
+    })
```
Comment (sttaran) [inline tests/locators/hw22.spec.js:8]: add navigation and opening popup to this hook

**Review by sttaran — CHANGES_REQUESTED** [review]

File: tests/locators/hw22.spec.js (hunk 1 above)
Comment (OlGubkina) [reply tests/locators/hw22.spec.js:66]: It was by mistake

File: tests/locators/hw22.spec.js (hunk 3 above)
Comment (OlGubkina) [reply tests/locators/hw22.spec.js:8]: fixed

File: tests/locators/hw22.spec.js (hunk 2 above)
Comment (OlGubkina) [reply tests/locators/hw22.spec.js:81]: fixed

File: tests/locators/hw22.spec.js [hunk 4]
```diff
@@ -0,0 +1,393 @@
...
+        test("Re-enter Password > Empty RepeatPassword", async ({page}) => {
+            const form = page.locator('app-signup-modal form')
+            const RepeatPasswordInput = form.locator('#signupRepeatPassword')
+
+            await RepeatPasswordInput.focus()
+            await RepeatPasswordInput.blur()
+
+            const RePasswordInputTip = page.locator('.invalid-feedback > p')
+            await expect(RePasswordInputTip).toContainText('Re-enter password required')
+            await expect(RePasswordInputTip).toHaveCSS('border-color', 'rgb(220, 53, 69)')
```
Comment (sttaran) [inline tests/locators/hw22.spec.js:364]: only classes should start with Upper letter

Comment (sttaran) [reply tests/locators/hw22.spec.js:364]: fix in all tests

**Review by sttaran — CHANGES_REQUESTED** [review]

**Pull request merged**